tests/test_parallel.py

```diff
@@ -43,9 +44,23 @@ def test_cache_cleared_by_other_process(
     extract("google.com")
     orig_unlink = os.unlink
 
-    def evil_unlink(filename: str) -> None:
+    def is_relative_to(path: Path, other_path: str | Path) -> bool:
+        """Return True if path is relative to other_path or False.
+
+        Taken from standard library (Python 3.9+ required).
```

Comment:
> Python 3.9+ required

This hunk works even though this project supports Python 3.8+?

Reply:
It's a [standard library method](https://docs.python.org/3/library/pathlib.html#pathlib.PurePath.is_relative_to) introduced in Python 3.9. So I had to implement it here as a function because Python 3.8 does not have it.

Reply:
Gotcha! I clarified the comment a bit.